Backend/router/v1/user.js

```diff
@@ -4,6 +4,9 @@ const models = require('../../models');
 const crypto = require('crypto');
 const session = require('express-session');
 
+let MIN_PASSWORD_LENGTH = 6;
+let MAX_PASSWORD_LENGTH = 16;
```

Comment:
Is there a reason we're limiting max to 16? Some of my passwords are longer than that, and I created then like them in purpose for security reasons.

Reply:
I was talking about max, not min.

Reply:
I modified password limitation as 8 to infinity